src/wizards/clusters.ts

```diff
@@ -126,7 +126,10 @@ export async function addClusterWizard(clusterSettings: ClusterSettings, explore
             saslOption,
         });
         explorer.refresh();
-        window.showInformationMessage(`Cluster '${name}' created successfully`);
+        const result = await window.showInformationMessage(`Cluster '${name}' created successfully`, 'Select', 'Close');

```

Comment:
I would simply select the created cluster. I think adding buttons bloats the UI

Reply:
You mean that explorer should be refreshed and select cluster must be done automatically? If it that it was my first idea but I cannot do that because:

 * vscode cannot reveal a treeview after a refresh (synchronization problem). See https://github.com/microsoft/vscode/issues/96089 (we should reopen this issue)
 * reveal an item means that it will force a connection. If you want to create several cluster without forcing connection, it will not possible. The Select UI button means, `do you want to open the cluster connection?`

Reply:
fixed with a setTimeout